x/mint/proto/mint/v1beta1/types.proto

```diff
@@ -7,72 +7,45 @@ import "gogoproto/gogo.proto";
 import "cosmos_proto/cosmos.proto";
 import "amino/amino.proto";
 
-// Minter represents the minting state.
-message Minter {
-  // current annual inflation rate
-  string inflation = 1 [
```

Comment:
fyi we are using this inflation parameter in the explorer, but I'm not sure if we still want to calculate a inflation rate on chain and update it frequently just for query purposes. 

cc @kpeluso

Reply:
yeah i deleted all that code

we can make some kind of workaround to support the explorer that's a read only query call

Reply:
alright I put it back, might want to check the math but yea